### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by l0ck3**
Makes the README nicer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
-# VOUS ETES TOUS DES MAUVAIS
-
+# VOUS ETES TOUS TRES BEAUX
```
Comment (BenjaminDigeon) [inline README.md]: CAPSLOCK

**Comment by l0ck3** [comment]
DOIGT!

**Comment by l0ck3** [comment]
Je fais ce que JE VEUX AVEC MES MAJUSCULES!

**Comment by l0ck3** [comment]
VA TE FAIRE MAJUSCULER!

File: README.md (hunk 1 above)
Comment (l0ck3) [reply README.md]: It's a title!

**Pull request merged**